include/deal.II/hp/dof_handler.h

```diff
@@ -627,6 +626,23 @@ namespace hp
     n_locally_owned_dofs_per_processor () const;
 
     /**
+     * Return an IndexSet describing the set of locally owned DoFs used for the
+     * given multigrid level. Since hp::DoFHandler does not support multilevel
+     * methods yet, this function returns an empty index set, IndexSet(0),
+     * independent of its argument.
+     */
+    const IndexSet &locally_owned_mg_dofs(const unsigned int level) const;
+
+    /**
+     * Return a vector that stores the locally owned level DoFs of each
+     * processor. Since hp::DoFHandler does not support multilevel methods yet,
+     * this function returns a vector of empty index sets, IndexSet(0),
```

Comment:
why not throw an error?

Reply:
That's what the other functions do, so I did not feel strongly about it. @bangerth you recently rewrote lots of `hp::DoFHandler`: What do you think?

Reply:
I think both functions should throw an `ExcNotImplemented()`. Would you mind making that change for both functions? Or to open a github issue?

Reply:
Definitely, I will add this assertion.